lynnsamuelson/LBMC-Hack
# El nodehost

What was changed:
I switched the inputs to bracket notation, because that is what the JSON parser is able to give back on the keys with spaces ("Contact ID" for example). 
I also npm-installed the cors package again, and set it up to run as middleware on all requests.


To test:
Make sure you have run `npm install` at the root of the project as well as in the lib folder.
Run `npm start` in the lib folder
Run `grunt` in the lib folder
Try to upload a file - it will give SQLITE errors and will console log pieces of info if you adjust the console log.
It will successfully pass the array of objects, but the issue is in the insert to db function -- if anyone wants to take a look, please do! I can still get my fake objects into the DB with the commented-out function, but the passed in ones must be missing something.
